pipelines/LSSTCam/nightly-validation.yaml

```diff
@@ -67,3 +67,7 @@ tasks:
     class: lsst.pipe.tasks.healSparseMapping.HealSparsePropertyMapTask
     config:
       connections.visit_summaries: preliminary_visit_summary
+  assembleCellCoadd:
+    class: lsst.drp.tasks.assemble_cell_coadd.AssembleCellCoaddTask
+    config:
+      connections.visitSummaryList: preliminary_visit_summary
```

Comment:
Do we want to generate and run measurement on cell-based coadds for NV?

Reply:
I'd lean towards doing that in a second phase, when we're ready to switch to it full in DRP; we don't want to be building two kinds of coadds in NV at any time, so we need to be ready for a complete switchover there.